pkgs/applications/networking/irc/srain/default.nix

```
@@ -0,0 +1,41 @@
+{ lib, stdenv, fetchFromGitHub, pkg-config, gtk3
+, libconfig, libsoup, libsecret, openssl, gettext
+, glib, glib-networking, appstream-glib, dbus-glib, gnome
+, python3, python3Packages # sphinx-build
+, meson, ninja, makeWrapper, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "srain";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "SrainApp";
+    repo = "srain";
+    rev = version;
+    sha256 = "14s0h5wgvlkdylnjis2fa7m835142jzw0d0yqjnir1wqnwmq1rld";
+  };
+
+  nativeBuildInputs = [
+    meson ninja pkg-config
+    gettext appstream-glib
+    makeWrapper wrapGAppsHook
+    python3Packages.sphinx
+  ];
+
+  buildInputs = [
+    gtk3 glib glib-networking dbus-glib
+    libconfig libsoup libsecret openssl
+    gnome.adwaita-icon-theme
+  ];
+
+  mesonBuildType = "release";
+
+  meta = with lib; {
+    description = "Modern IRC client written in GTK.";
+    homepage = "https://srain.im";
+    license = licenses.gpl3;
```

`gpl3` is a deprecated license, from [this](https://github.com/SrainApp/srain/blob/bfe78c88a06f6f5485634e26c3ff3b5a3e22c41b/src/core/app.c#L5-L8) it should be fine to use `gpl3Plus`.

```suggestion
    license = licenses.gpl3Plus;
```